src/snovault/elasticsearch/create_mapping.py

```diff
@@ -167,12 +169,13 @@ def index_settings():
         'settings': {
             'index.max_result_window': 99999,
             'index.mapping.total_fields.limit': 5000,
-            'index.number_of_shards': 5,
-            'index.number_of_replicas': 2,
+            'index.number_of_shards': 1,
+            'index.number_of_replicas': 0,
```

Comment:
Why are we lowering these?

Reply:
The old setting resulted in more than 1000 shards and hit the limit set by the Elasticsearch. The limit can be overridden but is not recommended. Regardless, this is changed in favor of [configurable shards](https://github.com/ENCODE-DCC/snovault/pull/295/files#diff-13be881525721a1251be911ff12f4c7a722ced10eb689a96d8c5821e8d9cd0e8L165) which also applies the same setting for single demos.

Reply:
Without the configurable shards branch, this new settings with 1 primary and 0 replicate will be applied in all cluster types. Old settings won't work in any cluster. With the configurable shards feature, local and single node demo clusters will have 1 primary and 0 replicate. Production cluster will have 3 primary and 1 replicate by default and is configurable at the deploy script.